From 9ab870520f839ea3d149e3db6400751556442dcc Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Fri, 13 Nov 2020 09:27:27 +0100
Subject: [PATCH] imx8mm-var-dart: Switch default dram size to 2G

When using CONFIG_DM_I2C to be able to interact
with i2c devices, early reading of the som eeprom
fails and only 512MB RAM is reported. Set DRAM
to 2G as this is the actual size.

Upstream-status: Inappropriate[configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 include/configs/imx8mm_var_dart.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/imx8mm_var_dart.h b/include/configs/imx8mm_var_dart.h
index 73dadcc83a..2e564369a1 100644
--- a/include/configs/imx8mm_var_dart.h
+++ b/include/configs/imx8mm_var_dart.h
@@ -210,7 +210,7 @@
 
 #define CONFIG_SYS_SDRAM_BASE           0x40000000
 #define PHYS_SDRAM                      0x40000000
-#define DEFAULT_SDRAM_SIZE		(512 * SZ_1M) /* 512MB Minimum DDR4, see get_dram_size */
+#define DEFAULT_SDRAM_SIZE		(2048 * SZ_1M) /* 512MB Minimum DDR4, see get_dram_size */
 #define VAR_EEPROM_DRAM_START          (CONFIG_SYS_MEMTEST_START + \
 					(DEFAULT_SDRAM_SIZE >> 1))
 
-- 
2.17.1

